Configuration/AlCa/python/autoCondPhase2.py

```diff
@@ -11,29 +11,32 @@
 #combines in a single dict of dict the tags defined below
 allTags={}
 
-
 allTags["LA"] = {
     'T6'  : ( ','.join( [ 'SiPixelLorentzAngle_phase2_T6_v2_mc'  ,SiPixelLARecord,connectionString, "", "2019-11-05 20:00:00.000"] ), ),  #uH = 0.106/T (TBPX), uH=0.0/T (TEPX+TFPX)
     'T14' : ( ','.join( [ 'SiPixelLorentzAngle_phase2_T14_v2_mc' ,SiPixelLARecord,connectionString, "", "2019-11-05 20:00:00.000"] ), ),  #uH = 0.106/T (TBPX), uH=0.0/T (TEPX+TFPX)
     'T15' : ( ','.join( [ 'SiPixelLorentzAngle_phase2_T15_v2_mc' ,SiPixelLARecord,connectionString, "", "2019-11-05 20:00:00.000"] ), ),  #uH = 0.053/T (TBPX), uH=0.0/T (TEPX+TFPX)
+    'T19' : ( ','.join( [ 'SiPixelLorentzAngle_phase2_T19_v1_mc' ,SiPixelLARecord,"frontier://FrontierPrep/CMS_CONDITIONS", "", "2019-12-20 14:00:00.000"] ), ),  #uH = 0.053/T (TBPX L3,L4), uH=0.0/T (TBPX L1,L2, TEPX+TFPX)
```

Comment:
@duartej I pushed the payloads to the preparation database instead of the production one as I thought my commits were meant for testing. As you are now pushing to CMSSW this must be changed to prod-DB.

Reply:
@duartej @mmusich I've uploaded the tags to ProdDB as well now.

Reply:
@mmusich thanks. Actually I was testing (trying the draft pull request buttom), but it turned out that all the tests started.. I wasn't sure about what I was doing :) My intention was to let @ghugo83 to do the PR as she kindly offered.

Reply:
@tvami, thanks! I suppose no further action is required from here, right?

Reply:
@duartej, wrong.
Please change all the `frontier://FrontierPrep/CMS_CONDITIONS` instances with `connectionString`

Reply:
@duartej In the current version of the file, the "FrontierPrep" is used, that should be changed back to "connectionString"; also the time stamp shall be something newer than 2 days ago (e.g. today)

Reply:
@mmusich, thanks , I'll do that

Reply:
@tvami, thanks for the info, I'm on it